rust/ql/test/library-tests/dataflow/models/main.rs

```diff
@@ -203,6 +203,16 @@ fn test_apply_flow_through() {
     sink(t); // $ hasValueFlow=33
 }
 
+async fn get_async_number(a: i64) -> i64 {
+    37
+}
```

Comment:
It would be nice to have a comment that this function has a summary model with taint through it.

Reply:
Makes sense. Done :)